internal/panda_controller/docker_agent.go

```diff
@@ -19,29 +19,42 @@ import (
 	"github.com/pkg/errors"
 )
 
-type dockerPandaAgent struct {
+type DockerPandaAgent struct {
 	grpcAgent   PandaAgent
 	cli         *docker.Client
 	containerId *string
 	sharedDir   *string
 }
 
-var _ PandaAgent = &dockerPandaAgent{}
+var _ PandaAgent = &DockerPandaAgent{}
+
+// See https://github.com/panda-re/panda/blob/dev/panda/python/core/pandare/qcows.json
```

Comment:
We should add this link/information from link somewhere that talks about the PANDA Config is we haven't already, and then update this to be a reference to that.